sdk/src/main/scala/com/horizen/block/WithdrawalEpochCertificate.scala

```diff
@@ -76,11 +75,8 @@ object WithdrawalEpochCertificate {
       currentOffset += o.size
     }
 
-    val nounce: Array[Byte] = BytesUtils.reverseBytes(certificateBytes.slice(currentOffset, currentOffset + 32))
-    currentOffset += 32
-
     new WithdrawalEpochCertificate(certificateBytes.slice(offset, currentOffset), version,
-      sidechainId, epochNumber, endEpochBlockHash, Array(), Array(), totalAmount, fee, quality, transactionOutputs, outputs)
+      sidechainId, epochNumber, endEpochBlockHash, Array(), totalAmount, fee, quality, transactionOutputs, outputs)
```

Comment:
@SergeyMakukha, the last version of `merge_support_branch` in zen core has different serialization:
https://github.com/ZencashOfficial/zen/pull/231/files#diff-35248149be7d2185b5cfa89c266e2da5R49
In general Certificate must be deserialized in the following order:
1. nVersion
2. scId
3. epochNumber
4. endEpochBlockHash
5. CTxIn vector
6. CTxOut vector
7. CBackwardTransferOut vector

Currently `quality` is not a parent of Certificate, so can be temporary hardcoded to some value, like 1.
Same for proof, can be hardcoded with zero-array with the length expected for zendoo.

Reply:
let leave as is, and change it asap as all field will be supported in MC

Reply:
ok. that's mostly for  @SergeyMakukha in STF branch with Certificate test case.